### PR TITLE
clutter-gst3: update to 3.0.27.

### PR DESCRIPTION
**Pull request by Cogitri**
* Enable gir

### CODE REVIEW

File: srcpkgs/clutter-gst3/template [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
+version=3.0.27
 revision=1
-wrksrc="clutter-gst-$version"
-configure_args="--disable-debug $(vopt_enable gir introspection)"
+wrksrc="clutter-gst-${version}"
 build_style=gnu-configure
-hostmakedepends="pkg-config intltool glib-devel $(vopt_if gir gobject-introspection)"
+builder_helper=gir
+configure_args="--disable-debug $(vopt_enable gir introspection)"
+hostmakedepends="pkg-config intltool glib-devel"
```
Comment (maxice8) [inline srcpkgs/clutter-gst3/template]: missing $(vopt_if vala vala)

File: srcpkgs/clutter-gst3/template [hunk 2]
```diff
@@ -1,24 +1,27 @@
 # Template file for 'clutter-gst3'
 pkgname=clutter-gst3
-version=3.0.26
+version=3.0.27
 revision=1
-wrksrc="clutter-gst-$version"
-configure_args="--disable-debug $(vopt_enable gir introspection)"
+wrksrc="clutter-gst-${version}"
 build_style=gnu-configure
```
Comment (maxice8) [inline srcpkgs/clutter-gst3/template]: quote

File: srcpkgs/clutter-gst3/template [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
-hostmakedepends="pkg-config intltool glib-devel $(vopt_if gir gobject-introspection)"
+builder_helper=gir
```
Comment (maxice8) [inline srcpkgs/clutter-gst3/template]: build_helper not builder_helper

Comment (maxice8) [reply srcpkgs/clutter-gst3/template]: and it should be quoted

**Comment by Cogitri** [comment]
Needs #8373

**Comment by Cogitri** [comment]
Rebased , let's see...

**Pull request closed without merging**